biz.aQute.bnd.reporter/test/biz/aQute/bnd/reporter/plugins/entries/bundle/GogoPluginTest.java

```diff
@@ -0,0 +1,49 @@
+package biz.aQute.bnd.reporter.plugins.entries.bundle;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import aQute.bnd.osgi.Jar;
+import aQute.bnd.osgi.Processor;
+import aQute.bnd.service.reporter.ReportEntryPlugin;
+import biz.aQute.bnd.reporter.plugins.serializer.JsonReportSerializerPlugin;
+import junit.framework.TestCase;
+
+public class GogoPluginTest extends TestCase {
+
+	public void testComponents() throws Exception {
+
+		final Jar jar = new Jar("jar", "testresources/gogoEntry/source.jar");
```

Comment:
You should close the jar object. Put it in a try-with-resource.

Reply:
done, thank you.